[DOP-8839] - add documentation to Samba connection (#153)
MobileTeleSystems/onetl: docs/changelog/next_release/150.feature.rst

Add ``Samba`` file connection.

> It is now possible to download and upload files to Microsoft Windows Server domains using base ``FileDownloader``/``FileUploader`` classes with Samba as ``connection`` parameter ("SMB" networking protocol by default).

Reviewer: ```suggestion
It is now possible to download and upload files to Samba shared folders using ``FileDownloader``/``FileUploader``.
```

Samba не обязательно на Windows, протокол может быть и NetBIOS.
